kaldb/src/test/java/com/slack/kaldb/writer/SpanFormatterTest.java

```diff
@@ -1,321 +0,0 @@
-package com.slack.kaldb.writer;
```

Comment:
SpanFormatterWithSchemaTest supercedes this. Moved a couple of useful tests from here to that class and then the rest are no longer valuable